default/customizations/custom_sitreps.txt

```diff
@@ -190,9 +190,41 @@ EffectsGroup
 //             ]
 //             empire = Source.Owner
 
+//#####     RANDOM BEGINNER HINTS     #####//
 
-
-
+EffectsGroup
+    scope = Source
+    activation = Turn low = 0 high = 200
```

Comment:
entries will repeat before the list is completely iterated.  The threshold could be lowered to turn 100 or so, or a different selection method used, though I think the hints should be once per turn (or in this case, an attempt once per turn).